kp_test.go

```diff
@@ -1601,10 +1623,16 @@ func TestSetAndGetMultipleInstancePolicies(t *testing.T) {
 		Enabled: true,
 		Network: "private-only",
 	}
+	intervalMonth := 2
```

Comment:
intervalMonth value here and on line 1605 do not match. Also this test is a bit weird, we input 4 policies but return 3 policies. Eventually we are just checking if the number of policies are greater than or equal to 0 which is why its passing. I believe this should be changed.

Reply:
Yes.
metrics policy block was missing in the old code, added. Thanks for pointing this out.
Also now I'm asserting with Equal instead of Greater.